code/game/jobs/job/command/cic/executive.dm

```diff
@@ -3,7 +3,7 @@
 	title = JOB_XO
 	flags_startup_parameters = ROLE_ADD_TO_DEFAULT|ROLE_ADMIN_NOTIFY
 	gear_preset = /datum/equipment_preset/uscm_ship/xo
-	entry_message_body = "<a href='"+URL_WIKI_XO_GUIDE+"'>You are second in command aboard the ship,</a> and are in next in the chain of command after the commanding officer. You may need to fill in for other duties if areas are understaffed, and you are given access to do so. Make the USCM proud!"
+	entry_message_body = "<a href='"+URL_WIKI_XO_GUIDE+"'>You are second in command aboard the USS Almayer,</a> and are in next in the chain of command after the Commanding Officer. Where applicable, you must abide by the <a href='"+URL_WIKI_CO_RULES+"'>Commanding Officer Code of Conduct</a>. You may need to fill in for other duties if areas are understaffed, and you are given access to do so. Make the USCM proud!"
```

Comment:
I believe we have a current_ship define somewhere which we should probably use instead of USS Almayer